setup.py

```diff
@@ -115,30 +116,40 @@ def user_dir():
 
 class DevelopCmd(develop):
     prefix_targets = [
-        ("nbconvert/templates", 'jdaviz-default'),
-        ("voila/templates", 'jdaviz-default')
+        (os.path.join("nbconvert", "templates"), 'jdaviz-default'),
+        (os.path.join("voila", "templates"), 'jdaviz-default')
     ]
 
     def run(self):
         target_dir = os.path.join(sys.prefix, 'share', 'jupyter')
         if '--user' in sys.prefix:  # TODO: is there a better way to find out?
             target_dir = user_dir()
         target_dir = os.path.join(target_dir)
+        is_win = sys.platform.startswith('win')
 
         for prefix_target, name in self.prefix_targets:
             source = os.path.join('share', 'jupyter', prefix_target, name)
             target = os.path.join(target_dir, prefix_target, name)
             target_subdir = os.path.dirname(target)
             if not os.path.exists(target_subdir):
                 os.makedirs(target_subdir)
-            rel_source = os.path.relpath(os.path.abspath(
-                source), os.path.abspath(target_subdir))
+            if not is_win:
+                rel_source = os.path.relpath(os.path.abspath(
+                    source), os.path.abspath(target_subdir))
+            else:  # relpath does not work if source/target on different Windows disks
+                rel_source = os.path.abspath(source)
```

Comment:
I don't fully grok what's happening here, but I have a suspicion that this isn't necessary if you use Paths. You can always switch between absolute and relative paths with `Path.absolute()` and `Path.relative_to(yourPath)`

Reply:
relative_to does not work if one is relative and the other one absolute, so you still need abs path for both to be safe.